src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -152,8 +193,8 @@ function DisabledReplyButton({
     >
       <Tooltip
         content={disabledMsg}
-        placement={"top"}
-        textAlign={"center"}
+        placement={Tooltip.Placement.TOP}
+        textAlign={Tooltip.Align.CENTER}
```

Comment:
You can actually leave those two as they are. We can discuss as to why, but Clever now prefers to just use strings instead of importing enums